examples/monoblock/simple_divertor_block.i

```diff
@@ -0,0 +1,594 @@
+#-------------------------------------------------------------------------
+# DESCRIPTION
+
+# Input file for computing the von-mises stress between the coolant pipe of a
+# tokamak divertor monoblock and its armour due to thermal expansion.
+# This simplified model is comprised of a solid OFHC copper cylinder surrounded
+# by tungsten armour; no interlayer is included and coolant flow is not
+# modelled. The boundary conditions are the stress-free temperature and the
+# block temperature to which the block is uniformly heated.
+# The solve is steady state and outputs temperature, displacement, and von
+# mises stress.
+
+#-------------------------------------------------------------------------
+# PARAMETER DEFINITIONS
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# File handling
+name=simple_divertor_block
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Geometry
+PI=3.141592653589793
+
+pipeIntDiam=12e-3    # m
+pipeExtDiam=15e-3    # m
+
+intLayerThick=1e-3   # m
+# intLayerIntDiam=${pipeExtDiam}
```

Comment:
Do the commented out lines serve a purpose or could they be deleted?

Reply:
These commented out line lines will be used in the full physics version, but are unused here in the simple version so I've removed them from this version of the input file.

On a related note, there are still more parameters than are technically required for this version (e.g. the cylinder diameter is given as the pipe outer radius plus the interlayer thickness despite this model having no interlayer) but I think it is worth leaving them as is for consistency in parameter names between models, which might be useful if working with both in parallel. It's also useful to convey how the simplified geometry is derived from the complex geometry.